libs/util/Collections.arucas

```diff
@@ -692,8 +692,8 @@ class BiMap: Map {
         if (this.containsKey(key) && value != this.get(key)) {
             throw new IllegalArgumentError("Cannot input duplcate key " + key + " for value : "+ value + ", use BiMap.forcePut(key, value) to replace!");
         }
-        if (this.containsValue(value) && key != this.getKey(value)) {
-            throw new IllegalArgumentError("Cannot input duplcate value " + value + " for key : "+ key + ", use BiMap.forcePut(key, value) to replace!");    
+        if (this.containsValue(value) && key != this.getKeys(value)) {
```

Comment:
This should be `getKey(value)` gets the key for a value (just a method that wraps `this.back.get(value)`.
`getKeys` is incorrect